src/main/java/ch/puzzle/eft/model/ExamFileModel.java

```diff
@@ -10,8 +13,16 @@ public ExamFileModel(File file) {
     }
 
     public String getSubjectName() {
-        return this.file.getParentFile()
-                        .getName();
+        if (this.file.getParentFile() != null) {
+            return this.file.getParentFile()
+                            .getName();
+        }
+        throw new ResponseStatusException(HttpStatus.NOT_FOUND,
```

Comment:
- Der Fehler soll an dieser Stelle geloggt werden, damit wir davon je erfahren.
- Der (englische) Fehlertext sollte etwa "No parent found for file {}" lauten.
- Die ResponseStatusException sollte einen 500 Fehler zurückgeben, weil dies ein Systemfehler ist. Der Fehlertext soll hier in Deutsch verfasst werden.